README.md

```diff
@@ -56,15 +56,15 @@ The core concepts of HTML, CSS and github were implemented.
 
 ## 🚀 Live Demo
 
-- [Live Demo Link](https://github.com/ehmaddd/mobile-first-portfolio/)
+- [Live Demo Link](https://github.com/ehmaddd/mobile-first-portfolio)
```

Comment:
- Hey, please, kindly use the Github pages to deploy your website as currently, you've added the link to the GitHub repo instead of the live demo link. Please, check the project requirement to know how to use the GitHub pages to deploy your app.